src/v2/Apps/Fair/FairSubApp.tsx

```diff
@@ -0,0 +1,49 @@
+import React from "react"
+import { AppContainer } from "v2/Apps/Components/AppContainer"
+import { createFragmentContainer, graphql } from "react-relay"
+import { FairSubApp_fair } from "v2/__generated__/FairSubApp_fair.graphql"
+import { Separator, Spacer } from "@artsy/palette"
+import { HorizontalPadding } from "v2/Apps/Components/HorizontalPadding"
+import { Footer } from "v2/Components/Footer"
+import { ErrorPage } from "v2/Components/ErrorPage"
+import { BackLink } from "v2/Components/Links/BackLink"
+
+interface FairAppProps {
+  fair: FairSubApp_fair
+}
+
+const FairApp: React.FC<FairAppProps> = ({ children, fair }) => {
+  if (!fair) return <ErrorPage code={404} />
+
+  return (
+    <>
+      <AppContainer>
+        <HorizontalPadding>
+          <BackLink
+            linkText={`Back to ${fair.name}`}
+            path={`/fair2/${fair.slug}`}
```

Comment:
This ship might have already sailed but it would be good to align our naming here with the larger router / idioms: `path` -> `to`

Reply:
Oh sorry-- I made this component about 2 PRs ago so we should feel free to rename. :D